README.md

```diff
@@ -2,32 +2,19 @@
 
-## Documentation
+`pytx3` is an all-in-one python package CLI which allows basic integration with ThreatExchange including matching on publically availible data.
 
-Full details on the ThreatExchange API and UI, data formats, and best practices are available in the [ThreatExchange docs](https://developers.facebook.com/docs/threat-exchange/).
+## Hashing Technologies
 
-## New tools for a new generation (2019 and beyond)
+ThreatExchange supports a variety of hashing algorithms for photos and videos. Two widely used algorithms are PDQ for photos and TMK for videos. The top level folders `pdq` and `tmk` includes documentation and  implementation exampels of these algorithms.
 
-In addition to the venerable pytx (see below), we now offer tag-based, descriptor-focused reference designs in [**Python**](https://github.com/facebook/ThreatExchange/blob/master/hashing/te-tag-query-python), [**Ruby**](https://github.com/facebook/ThreatExchange/blob/master/hashing/te-tag-query-ruby), and [**Java**](https://github.com/facebook/ThreatExchange/blob/master/hashing/te-tag-query-java).
+## Get All Available Data
 
-The common context to all these is that since early 2018 ThreatExchange has moved beyond malware/phishing into cross-company integrity-signal sharing. This newer tooling largely overlaps the old (such as pytx), but with an added focus on more interactive tooling for a broader, more diverse userbase; a strong threat-descriptor focus (vs malware analyses); and enhanced support for cross-company feedback mechanisms.
+For tag-driven workloads, supporting either bulk download or incremental updates, our currently recommended best practice is a [Java reference design](https://github.com/facebook/ThreatExchange/blob/master/api-reference-examples/java/te-tag-query/README.md).
 
-## Example code
-
-This project offers example code in Java, Python, PHP, and Ruby.  There's also a reference client-side user interface to ThreatExchange, written in JavaScript. (A far richer featureset is available in the hosted [TE UI](https://developers.facebook.com/docs/threat-exchange/ui).)
-
-## Get all available data
-
-For tag-driven workloads, supporting either bulk download or incremental updates, our currently recommended best practice is a [Java reference design](https://github.com/facebook/ThreatExchange/blob/master/hashing/te-tag-query-java/README.md).
-
-We also offer scripts in the `pytx/scripts` folder. For example, to get threat
-descriptors uploaded to ThreatExchange run the following command:
-
-```
-$ python pytx/scripts/get_data.py -o threat_descriptor
-```
+You can also explore the dataset using the hosted [ThreatExchange UI](https://developers.facebook.com/docs/threat-exchange/ui)
```

Comment:
Would move this into the API examples section or a specific note in the public documentation

Reply:
Moved to top level summary